### PR TITLE
docs: improve doc

### PR DESCRIPTION
**Pull request by apteles**

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
@apteles Thank you for contribution! For translations content we require to have at least one native speaker review. Could you assign someone to handle that?

**Comment by apteles** [comment]
You're welcome @alexandrtovmach . Of course I can. But I don't know how I could do that. Can you help me with any orientation?

**Comment by alexandrtovmach** [comment]
Just use `@mention` to ping native speaker

**Comment by alexandrtovmach** [comment]
@rogeriopradoj @walkeralencar could you help?

**Review by walkeralencar — CHANGES_REQUESTED** [review]
Tem uma regra de escrever o arquivo com 80 colunas, como foi feito no arquivo abaixo:
Fazendo este ajuste, estará em conformidade.

https://raw.githubusercontent.com/semver/semver.org/gh-pages/spec/v2.0.0.md

**Comment by walkeralencar** [comment]
> @rogeriopradoj @walkeralencar could you help?

In terms of translation, it's OK, but need to adjust the line limit to 80 cols to maintain code compatibility.

File: lang/pt-BR/index.md [hunk 1]
```diff
@@ -121,28 +121,35 @@ quando se determina a versão de precedência. Assim, duas versões que diferem
...
+      1. Identificadores consistindo apenas dígitos são comparados numericamente.
+
+      1. Identificadores com letras ou hífen são comparados lexicalmente na ordem de classificação ASCII.
+    
+      1. Identificadores numéricos sempre têm menor precedência do que os não numéricos.
+
+      1. Um conjunto maior de campos de pré-lançamento tem uma precedência maior do que
+           um conjunto menor, se todos os identificadores anteriores são iguais.
+
+        Exemplo: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
```
Comment (walkeralencar) [inline lang/pt-BR/index.md:149]: Need to respect the line limit of 80 cols.

**Comment by apteles** [comment]
That's fine, thank you for your review @walkeralencar. Anyway the project has a lint related to style guide and it has passed. The translate it's ok like @walkeralencar said. What you think about it, @alexandrtovmach?

**Review by walkeralencar — APPROVED** [review]
based on this comment: https://github.com/semver/semver.org/pull/292#issuecomment-714040770. It's approved.

**Comment by alexandrtovmach** [comment]
@walkeralencar Thanks for paying attention, but we're not support any length limitation anymore. It's turned off here: https://github.com/semver/semver.org/blob/gh-pages/.remarkrc#L8

**Comment by alexandrtovmach** [comment]
@apteles Thank you for contribution 🥇 and @walkeralencar for review 👍 
You're best!

**Pull request merged**